recommendation/microsoft.owin.security.json

```diff
@@ -0,0 +1,907 @@
+          "Actions": [
+            {
+              "Name": "ReplaceIdentifier",
+              "Type": "Identifier",
+              "Value": "HttpContext",
+              "Description": "Replace IAuthenticationManager with HttpContext",
+              "ActionValidation": {
+                "Contains": "HttpContext",
+                "NotContains": "IAuthenticationManager"
+              }
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "Microsoft.AspNetCore.Http",
+              "Description": "Add Microsoft.AspNetCore.Http namespace",
+              "ActionValidation": {
+                "Contains": "using Microsoft.AspNetCore.Http;",
+                "NotContains": ""
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Type": "Method",
+      "Name": "Microsoft.Owin.Security.IAuthenticationManager.AuthenticateAsync(string[])",
+      "Value": "Microsoft.Owin.Security.IAuthenticationManager.AuthenticateAsync(string[])",
+      "KeyType": "",
+      "ContainingType": "IAuthenticationManager",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Add Microsoft.AspNetCore.Authentication namespace if AuthenticationProperties is being used and remove Microsoft.Owin.Security.",
+          "Actions": [
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "Microsoft.AspNetCore.Authentication",
+              "Description": "Add Microsoft.AspNetCore.Authentication namespace",
+              "ActionValidation": {
+                "Contains": "using Microsoft.AspNetCore.Authentication;",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "ReplaceParametersOnly",
+              "Type": "Method",
+              "Value": "()",
+              "Description": "Remove parameters of AuthenticateAsync.",
+              "ActionValidation": {
+                "Contains": "",
+                "NotContains": ""
+              }
+            },
+            {
+              "Name": "AddComment",
+              "Type": "Method",
+              "Value": "This method does not take a list of string as a parameter, it can only accept a single scheme or no parameters.",
+              "Description": "Add a comment to explain how to use the parameters for the ChallengeAsync api.",
+              "ActionValidation": {
+                "Contains": "Please use your AuthenticationProperties object with this AuthenticateAsync api.",
```

Comment:
`Contains` field doesn't seem to match the value. Does this need to be addressed or am I missing something?

Reply:
Thank you, made me double check the rest of the validation as well and I caught another mismatch as well. Thanks :-)